lib/queuePopulator/QueuePopulator.js

```diff
@@ -350,7 +350,7 @@ class QueuePopulator {
                 break;
             default:
                 throw new Error("bad 'logSource' config value: expect 'bucketd,'" +
-                    ` mongo or 'dmd', got '${this.qpConfig.logSource}'`);
+                    ` mongo or 'kafka' or 'dmd', got '${this.qpConfig.logSource}'`);
```

Comment:
```suggestion
                throw new Error("bad 'logSource' config value: expect 'bucketd'," +
                    ` 'mongo', 'kafka' or 'dmd', got '${this.qpConfig.logSource}'`);
```

Reply:
Also the ticket says "_setupLogSources(): missing Kafka log source in the error we throw when no log source corresponds."
but it was here before right? Are we missing something?

Reply:
yes it is already in the error message, what was meant in the ticket is only that kafka was not listed as one of the expected values from what I understood